### PR TITLE
270 update readme

### LINKED ISSUE #270: Update README.md
- Remove reference to VT
- Add reference to BVH

### PR DESCRIPTION
**Pull request by uhetmaniuk**
Closes #270

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,9 +36,6 @@ Required for parallel execution of implicit time integration (quasi-statics).
 
 https://trilinos.org
 
-# VT
+## BVH
```
Comment (nmm0) [inline README.md]: Maybe we should have both? Since BVH is so heavily built on VT

**Review by nmm0 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (uhetmaniuk) [reply README.md]: There is no option for compiling with VT right now. 
It does not fit the sentence
"NimbleSM may be built with the following optional dependencies:"

Comment (uhetmaniuk) [reply README.md]: @nmm0 

I have updated the paragraph. Let me know what you think if you have time.

Comment (nmm0) [reply README.md]: Awesome thanks!

**Review by nmm0 — APPROVED** [review]
Looks good!

**Pull request merged**